### PR TITLE
Make descr more generic towards next majors

### PR DESCRIPTION
**Pull request by dereuromark**
There can only be one description in packagist etc. It should be version neutral.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -63,7 +63,6 @@
         }
     },
     "support": {
-        "email": "cakephp+csvview@josediazgonzalez.com",
```
Comment (ADmad) [inline composer.json]: If he wants to share his email leave it be :)

File: composer.json [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
     ],
-    "homepage": "http://github.com/friendsofcake/cakephp-csvview",
+    "homepage": "https://github.com/friendsofcake/cakephp-csvview",
     "license": "MIT",
     "authors": [
         {
             "name": "Jose Diaz-Gonzalez",
             "email": "email@josediazgonzalez.com",
-            "homepage": "http://josediazgonzalez.com",
+            "homepage": "https://josediazgonzalez.com",
```
Comment (Spriz) [inline composer.json]: This site doesn't work on HTTPS :P

Comment (dereuromark) [reply composer.json]: Geeze

Comment (josegonzalez) [reply composer.json]: It's on github pages and the ssl cert hasn't finished generating? Not sure what thats about...

File: composer.json (hunk 1 above)
Comment (josegonzalez) [reply composer.json]: I should really just remove my email from packages or change it to `email-for-consulting@josediazgonzalez.com`...

File: composer.json (hunk 2 above)
Comment (ADmad) [reply composer.json]: https://josediazgonzalez.com  is accessible now.

**Pull request merged**